### PR TITLE
feat(icons): icons prefix change

### PR DESCRIPTION
**Pull request by theJohnnyMe**
**Describe pull-request**  
Changing prefix in icons package/folder and in icon component

**Solving issue**  
Fixes: DTS-1694

**How to test**  
1. Check if the Gulp file in the icon package/folder has tds-icon as a prefix
2. Check if native and web component icon stories have tds as prefix 
3. Check if other components have native or web component with new prefix in place

**Suggested test steps**
- [ ] Browser testing (Chrome, Safari, Firefox) 
- [ ] Keyboard operability
- [ ] Interactive elements have labels.
- [ ] Storybook controls
- [ ] Design/controls/props is aligned with other components 
- [ ] Dark/light mode and variants 
- [ ] Input fields – values should be displayed properly 
- [ ] Events

**Additional context**  
Web font story will appear broken as new font files need upload to new CDN solution for Tegel icons.

### CODE REVIEW

File: core/src/components/icon/icon-font.stories.tsx [hunk 1]
```diff
@@ -46,17 +46,17 @@ export default {
...
 
-  <i class="sdds-icon ${args.icon}"></i>
+  <i class="tds-icon ${args.icon}"></i>
   `);
 
-export const Native = IconFontTemplate.bind({});
+export const WebFont = Template.bind({});
 
-Native.args = {};
+WebFont.args = {};
```
Comment (mJarsater) [inline core/src/components/icon/icon-font.stories.tsx:62]:
```suggestion
```

File: core/src/components/icon/icon.tsx [hunk 2]
```diff
@@ -5,12 +5,12 @@ import { Component, h, Prop, State, Host } from '@stencil/core';
...
 @Component({
-  tag: 'sdds-icon',
+  tag: 'tds-icon',
   styleUrl: 'icon.scss',
   shadow: true,
 })
 export class Icon {
-  /** Pass a name of the icon. For icon names, refer to https://digitaldesign.scania.com/foundations/icons/icon-library or storybook */
+  /** Pass a name of the icon. For icon names, refer to Storybook Icon controls dropdown or https://tegel.scania.com/foundations/icons/icon-library */
   @Prop({ reflect: true }) name: string = 'arrow_diagonal';
```
Comment (mJarsater) [inline core/src/components/icon/icon.tsx:14]: Should we have a default value?

File: core/src/components/popover-menu/popover-menu-item.scss [hunk 3]
```diff
@@ -29,7 +29,7 @@ ul.sdds-popover-menu-wrapper {
       display: flex;
       flex-wrap: nowrap;
 
-      i {
+      tds-icon {
```
Comment (mJarsater) [inline core/src/components/popover-menu/popover-menu-item.scss:32]: Im guessing we'll update the popover to use our component instead of the webfont. Nice

**Review by mJarsater — APPROVED** [review]
Nice! 🚀

File: core/src/components/popover-menu/popover-menu-item.scss (hunk 3 above)
Comment (theJohnnyMe) [reply core/src/components/popover-menu/popover-menu-item.scss:32]: In this story we had situation like this:
`<i>
   <tds-icon name="truck"></tds-icon>
</i>
`
so I dropped `<i> `and just left `<tds-icon>` and assigned styling rules directly to it. 
It seems that only in the button component we could choose between the web font and web component version of icon.

File: core/src/components/icon/icon.tsx (hunk 2 above)
Comment (theJohnnyMe) [reply core/src/components/icon/icon.tsx]: Truck it is ;)

**Pull request merged**